parsec/core/pki/submitter.py

```diff
@@ -63,7 +70,29 @@ async def new(
     async def submit(
         self, config_dir: Path, requested_device_label: DeviceLabel, force: bool
     ) -> "PkiEnrollmentSubmitterSubmittedCtx":
-        # TODO: document exceptions !
+        """
+        Raises:
+            BackendNotAvailable
+            BackendProtocolError
+
+            BackendCmdsInvalidRequest
+            BackendCmdsInvalidResponse
+            BackendNotAvailable
```

Comment:
Duplicated `BackendNotAvailable`

Funny enough, I don't see any reference to `BackendCmdsInvalidRequest`, `BackendCmdsInvalidResponse` and `BackendCmdsBadResponse` in the codebase (they are only referenced in `_send_cmd`'s documentation...). I think we should document `BackendConnectionError` instead, and add `BackendNotAvailable`/`BackendProtocolError`/`BackendOutOfBallparkError` sub errors (given they are useful to catch separatly)

Reply:
And it would be good to also correct `_send_cmd` ;-)